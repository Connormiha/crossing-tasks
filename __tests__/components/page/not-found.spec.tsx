import * as renderer from 'react-test-renderer';
import {BrowserRouter as Router} from 'react-router-dom';
import NotFoundPure from 'components/page/NotFound';

/* tslint:disable:jsx-wrap-multiline */
describe('<NotFoundPure />', () => {
    it('should render', () => {
        const tree = renderer.create(
            <Router>
                <NotFoundPure />
            </Router>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
