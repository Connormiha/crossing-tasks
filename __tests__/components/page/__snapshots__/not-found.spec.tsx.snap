// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPure /> should render 1`] = `
<div
  className="page-not-found"
>
  <h1
    className="page-not-found__title"
  >
    Page not found
  </h1>
</div>
`;
